src/commands/react-native/errors.ts

```diff
@@ -13,19 +12,21 @@ export class CodepushHistoryParseError extends Error {
 
 export class CodepushHistoryCommandError extends Error {
   constructor(message: string, command: string) {
+    let errorMessage: string
     try {
-      const commandMessage: string = JSON.parse(message).errorMessage
+      errorMessage = JSON.parse(message).errorMessage
```

Comment:
Because the [`constructor-super`](https://eslint.org/docs/latest/rules/constructor-super) eslint rule enforces to have a single `super()` call. The error says "Unexpected duplicate 'super()'".

Reply:
Ohh ack, thanks